build-scripts/generateRunners.ts

```diff
@@ -0,0 +1,66 @@
+/*!
+ * Copyright 2017 Amazon.com, Inc. or its affiliates. All Rights Reserved.
+ * SPDX-License-Identifier: MIT
+ */
+
+import * as fs from 'fs'
+import * as path from 'path'
+
+interface RunnerGenerator {
+    TaskName: string,
```

Comment:
Why PascalCase instead of camelCase? Same question applies to generate.json.

Reply:
All of the AWS js SDKs use PascalCase, so my brain was tainted by that, I can change it